style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  height: 100vh;
+  padding: 0 200px;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 150px 50px 150px;
+}
+
+header nav .logo a {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+header .toggle-button {
+  display: none;
+}
+
+header nav #hamburger-menu {
+  display: flex;
+}
+
+header nav #hamburger-menu li a {
+  display: block;
+  padding: 0 1rem;
+  padding-right: 0;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+header .head-body-container {
+  background-image: url(image/Header-bg.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 0 100px;
+}
+
+header .head-body {
+  padding: 500px 150px 380px 150px;
+}
+
+header .head-body h2 {
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
+  margin-bottom: 20px;
+}
+
+header .head-body p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+  margin-bottom: 20px;
+}
+
+header .head-body h3 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-bottom: 20px;
+}
+
+header .head-body ul {
+  display: flex;
+}
+
+header .head-body ul li {
+  margin-right: 30px;
+}
+
+@media (max-width: 500px) {
+  header {
```

Comment:
- Good job using media queries to make your site responsive across various screen sies, but I suggest that in order for your site to fit perfectly on all mobile screen sizes use a `max-width:765px`. Please fix this 👍 👍